wdl_runner/cromwell_launcher/Dockerfile

```diff
@@ -56,8 +56,8 @@ RUN chmod u+x /wdl_runner/wdl_runner.sh
 # Copy Cromwell and the Cromwell conf template
 RUN mkdir /cromwell
 RUN cd /cromwell && \
-    curl -L -O https://github.com/broadinstitute/cromwell/releases/download/0.19/cromwell-0.19.jar
-RUN ln /cromwell/cromwell-0.19.jar /cromwell/cromwell.jar
+    curl -L -O https://github.com/broadinstitute/cromwell/releases/download/0.19.3/cromwell-0.19.3.jar
```

Comment:
Thanks Paul. The last one tagged as a release, with release notes is 0.19.3, so that's the one I'm going with.